server/controllers/firestore.py

```diff
@@ -53,7 +50,12 @@ def _find(self, identifier, query=None):
         search_fields = identifier.keys()
         for field in search_fields:
             query = query.where(field, '==', identifier[field])
-        document = next(query.stream())
+        document = None
```

Comment:
```suggestion
```
Not strictly necessary, since you immediately assign to `document` before, but it doesn't really do any harm either.

Reply:
Removing this could have some weird scoping implications in different versions of python